<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="WKWZTPqvmdckJHX" failures="6" tests="15" name="com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest" time="254.180" errors="0" timestamp="2023-08-30T18:29:11 IST" skipped="0">
  <testcase classname="com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest" name="cookieButton" time="4.933"/>
  <system-out/>
  <testcase classname="com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest" name="verifyBreadcrumbOne" time="14.939"/>
  <system-out/>
  <testcase classname="com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest" name="verifyBreadcrumbTwo" time="13.081"/>
  <system-out/>
  <testcase classname="com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest" name="verifyBreadcrumbThree" time="7.548"/>
  <system-out/>
  <testcase classname="com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest" name="verifyHyperLinkOne" time="10.425">
    <failure message="expected [treating-cold-flu/is-feed-a-cold-starve-a-fever-true/] but found [https://www.theraflu.com/treating-cold-flu/is-feed-a-cold-starve-a-fever-true/]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [treating-cold-flu/is-feed-a-cold-starve-a-fever-true/] but found [https://www.theraflu.com/treating-cold-flu/is-feed-a-cold-starve-a-fever-true/]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.baseClass.BaseClass.compareTwoText(BaseClass.java:831)
at com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest.verifyHyperLinkOne(TherafluColdAndFluWhatsTrueAndFalseTest.java:84)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyHyperLinkOne -->
  <system-out/>
  <testcase classname="com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest" name="verifyHyperLinkTwo" time="9.649">
    <failure message="expected [prevention/antibiotics-dont-work-for-everything/] but found [https://www.theraflu.com/prevention/antibiotics-dont-work-for-everything/]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [prevention/antibiotics-dont-work-for-everything/] but found [https://www.theraflu.com/prevention/antibiotics-dont-work-for-everything/]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.baseClass.BaseClass.compareTwoText(BaseClass.java:831)
at com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest.verifyHyperLinkTwo(TherafluColdAndFluWhatsTrueAndFalseTest.java:96)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyHyperLinkTwo -->
  <system-out/>
  <testcase classname="com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest" name="verifyHyperLinkThree" time="13.631">
    <failure message="expected [treating-cold-flu/how-the-flu-shot-works/] but found [https://www.theraflu.com/treating-cold-flu/how-the-flu-shot-works/]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [treating-cold-flu/how-the-flu-shot-works/] but found [https://www.theraflu.com/treating-cold-flu/how-the-flu-shot-works/]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.baseClass.BaseClass.compareTwoText(BaseClass.java:831)
at com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest.verifyHyperLinkThree(TherafluColdAndFluWhatsTrueAndFalseTest.java:108)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyHyperLinkThree -->
  <system-out/>
  <testcase classname="com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest" name="verifyHyperLinkFour" time="11.097">
    <failure message="expected [prevention/healthy-chicken-soup-recipes/] but found [https://www.theraflu.com/prevention/healthy-chicken-soup-recipes/]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [prevention/healthy-chicken-soup-recipes/] but found [https://www.theraflu.com/prevention/healthy-chicken-soup-recipes/]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.baseClass.BaseClass.compareTwoText(BaseClass.java:831)
at com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest.verifyHyperLinkFour(TherafluColdAndFluWhatsTrueAndFalseTest.java:120)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyHyperLinkFour -->
  <system-out/>
  <testcase classname="com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest" name="verifyReferences" time="8.385"/>
  <system-out/>
  <testcase classname="com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest" name="verifyRecommendedArticlesOne" time="68.090"/>
  <system-out/>
  <testcase classname="com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest" name="verifyRecommendedArticlesTwo" time="32.695"/>
  <system-out/>
  <testcase classname="com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest" name="verifyRecommendedArticlesThree" time="14.472"/>
  <system-out/>
  <testcase classname="com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest" name="verifyRecommendedArticlesFour" time="13.499"/>
  <system-out/>
  <testcase classname="com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest" name="verifyHyperLinkFive" time="16.818">
    <failure message="expected [products/all/] but found [https://www.theraflu.com/products/all/]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [products/all/] but found [https://www.theraflu.com/products/all/]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.baseClass.BaseClass.compareTwoText(BaseClass.java:831)
at com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest.verifyHyperLinkFive(TherafluColdAndFluWhatsTrueAndFalseTest.java:191)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyHyperLinkFive -->
  <system-out/>
  <testcase classname="com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest" name="verifyHyperLinkSix" time="14.918">
    <failure message="expected [products/multi-symptom-severe-cold-tea-infusions/] but found [https://www.theraflu.com/products/multi-symptom-severe-cold-tea-infusions/]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [products/multi-symptom-severe-cold-tea-infusions/] but found [https://www.theraflu.com/products/multi-symptom-severe-cold-tea-infusions/]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.baseClass.BaseClass.compareTwoText(BaseClass.java:831)
at com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest.verifyHyperLinkSix(TherafluColdAndFluWhatsTrueAndFalseTest.java:203)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyHyperLinkSix -->
  <system-out/>
</testsuite> <!-- com.qa.theraflu.test.TherafluColdAndFluWhatsTrueAndFalseTest -->
